openapiart/tests/test_set.py

```diff
@@ -0,0 +1,117 @@
+import sys
+import os
+import pytest
+
+sys.path.append(os.path.join(os.path.dirname(__file__), "..", "..", "art"))
+import sanity
+
+
+def test_set(api):
+    config = sanity.Api().prefix_config()
+
+    # Use the new set API to set normal property values of each class
+    config.set(
+        ieee_802_1qbb=True,
+        space_1=10,
+        full_duplex_100_mb=20,
+        response=config.STATUS_200,
+        a="abc",
+        b=0.2,
+        c=20,
+        d_values=[config.A],
+        h=False,
+        i="1010100",
+        list_of_string_values=["abc", "efg"],
+        list_of_integer_values=[1, 2, 3, 4],
+    )
+    config.e.set(e_a=1.1, e_b=2.1)
+    config.l.set(
+        ipv4="10.1.1.1",
+        ipv6="::",
+        mac="aa:bb:cc:11:22:33",
+        string_param="abcd",
+    )
+    # Use the new set API to set pattern property values of each class
+    config.mac_pattern.mac.increment.set(
+        start="aa:aa:bb:bb:cc:cc", step="00:01:00:00:10:00", count=10
+    )
+    config.integer_pattern.integer.decrement.set(start=200, step=2, count=100)
+
+    config_data = config._properties
+
+    if not config_data["ieee_802_1qbb"]:
```

Comment:
I guess instead of a `if` check it would better we call `assert` for verifying the parameters.
I see rest of the unit test cases use assert and it will good to follow a common approach for the whole test directory
for example
```python
assert config_data["ieee_802_1qbb"] == True, "ieee_802_1qbb value is not set correctly"
``` 
for more reference you can follow the link 
https://www.geeksforgeeks.org/python-assert-keyword/

Reply:
yup, assert is more idiomatic for tests

Reply:
changed if to assert